### PR TITLE
Fix Github security warning

### PR DESCRIPTION
**Pull request by ftclausen**
Also use package-lock.json to make builds more reproducible.

This is not quite ready to merge due to a failed test:

```
    Window size
      ✗ resizes the window
        - Expected 296 to equal 200.
            at /Users/fclausen/bbgit/protractor-sync/test/spec/protractor_sync_test.ts:805:30
            at /Users/fclausen/bbgit/protractor-sync/test/spec/protractor_sync_test.ts:46:7
            at fiberContents (/Users/fclausen/bbgit/protractor-sync/node_modules/asyncblock/lib/asyncblock.js:92:26)
```

I have not yet been able to fix it so opening it up to you guys to have a look in case you have any suggestions.

UPDATE: Works on CircleCI - this was something on my local machine.

### CODE REVIEW

**Comment by mindywhitsitt** [comment]
I'm confused - I seem to be seeing it fail on the `sudo apt-get update` command, not on any actual test. Maybe I'm not reading the CircleCI output correctly?

**Comment by ftclausen** [comment]
> I'm confused - I seem to be seeing it fail on the `sudo apt-get update` command, not on any actual test. Maybe I'm not reading the CircleCI output correctly?

Ah, sorry, going by what was happening locally and thought it would get at least past the set up steps here. I'll debug.

**Comment by mindywhitsitt** [comment]
Maybe once it gets past the setup steps on the build server, then the test will also pass. :)

**Comment by ftclausen** [comment]
Ah, the issue is our Node is too old and that means the Circle CI image is so old Debian no longer mirrors the files. I'll see if we can just bump to the latest LTS and see if things keep working

**Comment by ftclausen** [comment]
> Maybe once it gets past the setup steps on the build server, then the test will also pass. :)

You were right, once I used current Node LTS + matching container image the build worked fine. The test passes on CircleCI so must be something on my machine.

**Review by mindywhitsitt — APPROVED** [review]
Ok cool. Well, it would still be nice to know what's happening on your machine, but passing on the build server sounds good enough for an approval if you ask me (which you did). ;)

**Comment by ftclausen** [comment]
Yeah, I still plan to debug this local issue and we can also see what the other reviewers have to say.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "protractor-sync",
-  "version": "6.0.2",
+  "version": "6.0.3",
   "description": "Protractor 5.x wrapper adding synchronous, polled selectors and expectations",
```
Comment (wesmcouch) [inline package.json:4]: We might update this description to remove the version

**Review by wesmcouch — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (mindywhitsitt) [reply package.json:4]: Hm. Why?

Comment (wesmcouch) [reply package.json:4]: Hah completely misread this, I thought it said Protractor Sync 5.x and was referencing itself.

**Pull request closed without merging**